arches/app/search/components/base.py

```diff
@@ -83,10 +90,8 @@ def get_filter(self, componentname):
     def get_searchview_name(self):
         if not self.request:
             searchview_component_name = None
-        elif self.request.method == "POST":
-            searchview_component_name = self.request.POST.get("search-view", None)
         else:
-            searchview_component_name = self.request.GET.get("search-view", None)
+            searchview_component_name = self.search_request.get("search-view", None)
```

Comment:
I think we only need the "else" portion of this statement.  self.request is already assumed to exist in the __init__ method.  If it doesn't that method will fail.  Will self.request ever be None?